Makefile

```diff
@@ -7,6 +7,7 @@ up:
 	pip install pipenv
 	pipenv install
 	bundle install
+	npm --prefix ./build ci
```

Comment:
never used `--prefix` before but it seemed like the place to put it (given the Ruby & Python in `build/` as well)